hs/src/Reach/Compiler.hs

```diff
@@ -22,6 +22,7 @@ import Reach.Eval
 import Reach.Linearize
 import Reach.Optimize
 import Reach.Parser (gatherDeps_top)
+import Reach.SimServer
```

Comment:
I think you should make a `Simulator` directory with `Server` and `Eval` (or `Core` or something)